client/homebrew/brewRenderer/brewRenderer.jsx

```diff
@@ -186,12 +186,14 @@ const BrewRenderer = (props)=>{
 		} else {
 			if(pageText.startsWith('\\page')) {
 				const firstLineTokens  = Markdown.marked.lexer(pageText.split('\n', 1)[0])[0].tokens;
-				const injectedTags = firstLineTokens.find((obj)=>obj.injectedTags !== undefined)?.injectedTags;
```

Comment:
What happens if you just use conditional chaining here:

```
injectedtags = firstLineTokens?.find((...

or

(obj)=>obj?.injectedTags ...
```

I believe one of the above should be basically the same result as this PR, but not require nesting `if` statements.

Reply:
It is, I just prefer blocks as more obvious  - even if less elegant. :)

Switched it out for your solution.